src/backend/utils/__mocks__/indexer.js

```diff
@@ -20,8 +27,22 @@ const deletePost = (postId) => {
   return Promise.resolve();
 };
 
-const search = (keyword = '') => {
-  const filtered = db.values.filter((value) => value.text.includes(keyword));
+const mockCreateFieldsFromFilter = (filter) => {
+  switch (filter) {
+    case 'author':
+      return ['author'];
+    case 'post':
+    default:
+      return ['text', 'title'];
+  }
+};
+
+const search = (keyword = '', filter = 'post') => {
```

Comment:
This still doesn't match what you're doing in the other file.  It needs to be identical:

```
const search = async (textToSearch, filter = 'post') => {
```